{
    "created": 1730995908.7134912,
    "duration": 0.031216144561767578,
    "exitcode": 2,
    "root": "/tmp/code_fk6m403l",
    "environment": {},
    "summary": {
        "total": 0,
        "collected": 0
    },
    "collectors": [
        {
            "nodeid": "",
            "outcome": "passed",
            "result": [
                {
                    "nodeid": "test_enigma.py",
                    "type": "Module"
                }
            ]
        },
        {
            "nodeid": "test_enigma.py",
            "outcome": "failed",
            "result": [],
            "longrepr": "ImportError while importing test module '/tmp/code_fk6m403l/test_enigma.py'.\nHint: make sure your test modules/packages have valid Python names.\nTraceback:\n/home/brad/.miniconda3/lib/python3.10/importlib/__init__.py:126: in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\ntest_enigma.py:4: in <module>\n    from EnigmaModel import EnigmaModel\nE   ImportError: cannot import name 'EnigmaModel' from 'EnigmaModel' (/tmp/code_fk6m403l/EnigmaModel.py)"
        }
    ],
    "tests": []
}